.run/production-debug.run.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectRunConfigurationManager">
-    <configuration default="false" name="production-debug" type="FlutterRunConfigurationType" factoryName="Flutter">
+    <configuration name="production-debug" default="false" factoryName="Flutter"
+        type="FlutterRunConfigurationType">
```

Comment:
Is this staging?

Reply:
@diarmaidlindsay 
I think we should only do development and production environments